object_database/web/cells/cells.py

```diff
@@ -2547,7 +2710,7 @@ def recalculate(self):
 class Clickable(Cell):
     def __init__(self, content, f, makeBold=False, makeUnderling=False):
         super().__init__()
-        self.f = f
+        self.f = f  # What is this?
```

Comment:
it should be called 'onClick'.  It performs the button action when clicked. If it returns a string, then we navigate to the specified url. See #128 for more on this, BTW.